rapid7_insightidr/plugin.spec.yaml

```diff
@@ -616,31 +646,56 @@ actions:
         type: string
         example: "where(user=adagentadmin, loose)"
         required: true
+        order: 1
+      relative_time:
+        title: Relative Time
+        description: A relative time in the past to look for alerts
+        type: string
+        example: "Last 5 Minutes"
+        default: "Last 5 Minutes"
+        required: true
+        order: 2
+        enum:
+          - "Last 5 Minutes"
+          - "Last 10 Minutes"
+          - "Last 20 Minutes"
+          - "Last 30 Minutes"
+          - "Last 45 Minutes"
+          - "Last 1 Hour"
+          - "Last 2 Hours"
+          - "Last 3 Hours"
+          - "Last 6 Hours"
+          - "Last 12 Hours"
+          - "Use Time From Value"
       time_from:
         title: Time From
-        description: Beginning time and date for the query. The format is flexible and will work with simple dates (e.g. 01-01-2020) to full ISO time (e.g. 01-01-2020T00:00:00)
+        description: Beginning date and time for the query. This will be ignored unless relative time is set to 'Use Time From Value'. The format is flexible and will work with simple dates (e.g. 01-01-2020) to full ISO time (e.g. 01-01-2020T00:00:00)
```

Comment:
```suggestion
        description: Beginning date and time for the query. This will be ignored unless Relative Time input is set to 'Use Time From Value'. The format is flexible and will work with simple dates (e.g. 01-01-2020) to full ISO time (e.g. 01-01-2020T00:00:00)
```